src/main/java/core/basesyntax/model/machine/Machine.java

```diff
@@ -1,9 +1,29 @@
 package core.basesyntax.model.machine;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+
+@Entity
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
 public class Machine {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
this inheritance strategy required a different id generation type

Reply:
corrected